src/modules/emote_menu/twitch/EmoteMenu.jsx

```diff
@@ -91,18 +109,25 @@ export default class EmoteMenuModule {
     }
 
     if (isMounted) {
-      this.show(emoteMenuEnabled);
+      this.show(emoteMenuValue);
     }
   }
 
-  show(visible) {
+  show(emoteMenuValue) {
     const legacyContainer = document.querySelector(LEGACY_BTTV_EMOTE_PICKER_BUTTON_CONTAINER_SELECTOR);
-
     if (legacyContainer == null) {
```

Comment:
this logic doesn't make sense. how is the legacy container not null when we want to to show the new button?